pkg/proxy/apis/kubeproxyconfig/v1alpha1/types.go

```diff
@@ -148,14 +148,11 @@ type KubeProxyConfiguration struct {
 	ConfigSyncPeriod metav1.Duration `json:"configSyncPeriod"`
 }
 
-// Currently two modes of proxying are available: 'userspace' (older, stable) or 'iptables'
-// (newer, faster). If blank, use the best-available proxy (currently iptables, but may
-// change in future versions).  If the iptables proxy is selected, regardless of how, but
-// the system's kernel or iptables versions are insufficient, this always falls back to the
-// userspace proxy.
+// Currently, four modes of proxying are available total: 'userspace' (older, stable), 'iptables'
```

Comment:
kube-proxy should break into 2 binaries - linux and windows - the flags and docs are so totally disjoint.

Reply:
I fixed it now. PTAL.